### PR TITLE
[11.0][account_move_batch_validate]. Add try..except during import of queue_job

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Otherwise anyone using this repo will need to install the queue_job, if they add the repo in their addons-path.

cc @grindtildeath @benwillig

### CODE REVIEW

File: account_move_batch_validate/wizard/account_move_validate.py [hunk 1]
```diff
@@ -2,8 +2,22 @@
...
+import logging
+
 from odoo import api, fields, models
-from odoo.addons.queue_job.job import job
+
+_logger = logging.getLogger(__name__)
+
+
+try:
+    from odoo.addons.queue_job.job import job, Job
```
Comment (grindtildeath) [inline account_move_batch_validate/wizard/account_move_validate.py]: Remove `Job` to make travis happy.
```suggestion
    from odoo.addons.queue_job.job import job
```

**Review by grindtildeath — COMMENTED** [review]
Looks legit, but travis is raising an error:
`account_move_batch_validate/wizard/account_move_validate.py:13:5: F401 'odoo.addons.queue_job.job.Job' imported but unused`

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
@grindtildeath good to go?

**Review by grindtildeath — APPROVED** [review]
@lreficent definitely

**Pull request merged**